app/helpers/media_helper.rb

```diff
@@ -1,13 +1,11 @@
 module MediaHelper
   def media_thumbnail(media_item)
-    if media_item.thumbnail?
-      if media_item.caption.text.present?
-        alt_text = truncate(media_item.caption.text, length: 36)
-        return image_tag(media_item.item.thumb.url, alt: alt_text)
-      else
-        # Use auto-generated alt text
-        return image_tag(media_item.item.thumb.url)
-      end
+    if media_item.thumbnail? && media_item.caption
+      # Use auto-generated alt text if no caption text
+      return image_tag(media_item.item.thumb.url) unless media_item.caption.text.blank?
```

Comment:
We should be returning the image tag as is without a user-generated caption if no caption exists. This current item would only run this line if the caption was not blank, which is the opposite of what we want.

Reply:
Maybe use something like this psuedocode:

```
    if media_item.thumbnail?
      if media_item.caption? (and possibly && media.caption.text?)
        alt_text = truncate(media_item.caption.text, length: 36)
        return image_tag(media_item.item.thumb.url, alt: alt_text)
      else
        # For images with no captions
        # Use auto-generated alt text
        return image_tag(media_item.item.thumb.url)
      end
```